index.html

```diff
@@ -1,10 +1,73 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
+    <meta name="description" content="homework" />
+    <meta name="keywords" content="lists tables and forms" />
+    <meta name="author" content="Alexey Tretyakov" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/style.css">
     <title>lesson</title>
 </head>
 <body>
-<h1>lesson 1</h1>
+<div class="advantages">
```

Comment:
**Semantic recommendation** 
It's better to use `<section> ` here.
The` <section>` tag defines a section in a document.
![image](https://user-images.githubusercontent.com/25488411/156405622-de79942f-de2a-4d42-8651-6de5a089d580.png)